systemtest/jpsreport_tests/CMakeLists.txt

```diff
@@ -17,7 +26,16 @@ set_tests_properties(
   05_method_A
   01_method_B
   01_method_C
-  01_method_D
+        D01_test_reference_data
```

Comment:
please use the same indention

Reply:
Done. Happened when automatically reformating the code...